src/main/java/com/conveyal/taui/analysis/RegionalAnalysisManager.java

```diff
@@ -0,0 +1,141 @@
+package com.conveyal.taui.analysis;
+
+import com.amazonaws.regions.Region;
+import com.amazonaws.regions.Regions;
+import com.amazonaws.services.s3.AmazonS3;
+import com.amazonaws.services.s3.AmazonS3Client;
+import com.amazonaws.services.sqs.AmazonSQS;
+import com.amazonaws.services.sqs.AmazonSQSClient;
+import com.conveyal.r5.analyst.broker.JobStatus;
+import com.conveyal.r5.analyst.cluster.GridRequest;
+import com.conveyal.r5.analyst.cluster.GridResultAssembler;
+import com.conveyal.r5.analyst.cluster.GridResultConsumer;
+import com.conveyal.r5.analyst.scenario.Scenario;
+import com.conveyal.r5.profile.ProfileRequest;
+import com.conveyal.taui.AnalystConfig;
+import com.conveyal.taui.models.Bundle;
+import com.conveyal.taui.models.Project;
+import com.conveyal.taui.models.RegionalAnalysis;
+import com.conveyal.taui.persistence.Persistence;
+import com.conveyal.taui.util.JsonUtil;
+import com.mashape.unirest.http.Unirest;
+import com.mashape.unirest.http.exceptions.UnirestException;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import java.io.ByteArrayOutputStream;
+import java.io.File;
+import java.io.IOException;
+import java.io.Serializable;
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Map;
+import java.util.concurrent.ArrayBlockingQueue;
+import java.util.concurrent.ThreadPoolExecutor;
+import java.util.concurrent.TimeUnit;
+
+/**
+ * Manages coordination of multipoint runs with the broker.
+ */
+public class RegionalAnalysisManager {
+    private static final Logger LOG = LoggerFactory.getLogger(RegionalAnalysisManager.class);
+    private static AmazonS3 s3 = new AmazonS3Client();
+
+    private static ThreadPoolExecutor executor = new ThreadPoolExecutor(1, 20, 90, TimeUnit.SECONDS, new ArrayBlockingQueue<>(512));
+
+    public static Map<String, JobStatus> statusByJob;
+
+    public static final GridResultConsumer consumer;
+    public static final String resultsQueueUrl;
+
+    static {
+        AmazonSQS sqs = new AmazonSQSClient();
+        sqs.setRegion(Region.getRegion(Regions.fromName(AnalystConfig.region)));
+        resultsQueueUrl = sqs.getQueueUrl(AnalystConfig.resultsQueue).getQueueUrl();
+        consumer = new GridResultConsumer(resultsQueueUrl, AnalystConfig.resultsBucket);
+
+        new Thread(consumer, "queue-consumer").start();
+    }
+
+    public static void enqueue (RegionalAnalysis regionalAnalysis) {
+        String brokerUrl = AnalystConfig.offline ? "http://localhost:6001" : AnalystConfig.brokerUrl;
```

Comment:
Why not just force AnalystConfig.brokerUrl to the local URL when starting up, if offline = true?